### ISSUE TITLE
Repeatable Job : "every" configuration and its behavior

### ISSUE THREAD

**Issue by loicsimon** [taskforcesh/bullmq#899]
Hello,

First of all : thank you for your library

I'm trying to use repeatable jobs like this :
 - create a job which will run in one hour
 - for 24h run it every hours

I create this repeatable job with the following options :

```json
{
    "every": 3600000,
    "immediately": false,
    "limit": 24,
}
```

The problem is that my job isn't scheduled in one hour but on the next hour.
Example :
Right now it's 23:41. I expect my job to be scheduled at 00:41 but it's scheduled at 00:00

If I set `immediately` to `true`, then as expected the first job is run immediately and the next one is scheduled at 00:41.

It looks like this inconsistent behavior comes from these lines in [repeat.ts](https://github.com/taskforcesh/bullmq/blob/master/src/classes/repeat.ts#L233) :
```ts
if (opts.every) {
    return (
      Math.floor(millis / opts.every) * opts.every +
      (opts.immediately ? 0 : opts.every)
    );
```
Why not use `millis` without rounding it? Is it to have the same behavior than `cron` configuration?

How can I get my expected behavior ?

Thank you.

**Comment by oanguenot** [comment]
Hi, I have the same request.
In my case, I would like to avoid peaks of run (due to the jobs "rounded" time) 
I use BullMQ 2.1.1 with Redis 6.0

Example If I created repeatable jobs with different names at 8:21, 8:23 and 8:24 which should run every hour, I would like that the next runs are 9:21, 9:23 and 9:24 and then 10:21, 10:23 and 10:24 instead of 9:00 for the 3 jobs and then 10h00 for the next iteration.

Is there a way to do that ?

I tested with immediately, but it seems that all other jobs with the same "every" value are executed at the same schedule created by the first job (whatever the name). So, it is not the behavior expected.

Thanks in advance
